field/numeric_test.go

```diff
@@ -120,3 +120,32 @@ func TestNumericSetBytesSetsDataOntoDataStruct(t *testing.T) {
 
 	require.Equal(t, 9, data.Value)
 }
+
+func TestNumericJSONMarshal(t *testing.T) {
+	numeric := NewNumeric(&Spec{
```

Comment:
I'm thinking if we can repalce it with:
```go
numeric := NewNumericValue(1)
```

It seems we don't do pack/unpack during marshaling and we may not need Spec here.

Reply:
nice find! I'll do the same for strings and binary too